concepts/observability-strategy/configurable-logging/01-logging-groundwork.md

```diff
@@ -0,0 +1,48 @@
+Furthermore, Loki is provided in a non-scalabale setup and cannot be configured at runtime.
+
+As outlined in the [general strategy](../strategy.md), integration (and with that, changing the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can simply bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few.
+
+This concept proposes how to open up to those new scenarios by supporting convenient integration at runtime.
+
+## Requirements
+
+### Basic backend configuration
+- Have a vendor-neutral layer of collectors that collects and ships the telemetry data, but does not permanently store it (as a backend)
+- Support configuration of the selected collector at runtime (no need to run a Kyma upgrade process) in a scenario-focused approach
+- Support multiple configurations at the same time in individual resources to enable easy activation of dedicated scenarios
+- As a minimum, support one vendor-neutral input and output. It should be possible to chain your custom collector for specific conversions. For example, for traces, supporting the OLTP protocol will support most of the vendors already. Chaining a custom OpenTelemetry  Collector can do custom conversion to a specific protocol.
+- Support all typical settings for the supported outputs of the used collector, do not hide or abstract them
+- The collector must run stable at any time. Bad configuration must be prevalidated and rejected. Fast feedback is welcome.
+- Secrets must be kept secret.
+- Scenarios/Pipelines need to be isolated and have it's own buffer management. If a backend is in a bad shape and cannot process any data anymore, data should still continue to be pushed to other backends
+- Typical auth mechanisms for the integration must be support, especially also client certificate based solutions
+- For logging: de-dotting for elasticsearch must be possible
+- Filtering of unrelevant data (like dropping logs of kyma-system Namespace) must be possible
+
+### Template definitions
+- Have a mechanism to provide templates and best practices for typical scenarios, which can be instantiated at runtime
+- Such template provides the same feature-richness as a configuration scenario
+- If specific templates are bundled with Kyma, updates of a template in use must propagate to the actual collector configuration
+  - Support placeholder definitions with default values and descriptions
+
+### Template instantiation
+- A template is instantiated by binding it to a secret that provides input for placeholders like URL and credentials. Alternatively, placeholders can be filled out with configMaps and static values.
+  - The template validates whether all placeholders will be replaced with values.
```

Comment:
```suggestion
  - The template validates whether all placeholders have been replaced with values.
```
It can only validate that after it happened, or am I misunderstanding something here?